src/i18n/en.ts

```diff
@@ -182,7 +182,7 @@ const en = {
     "asset.create.hideAdvancedSection": "Hide advanced options",
     "asset.remove.tooltip": "Remove this asset",
     "asset.remove.dialog.title": 'Remove {type} "{label}"?',
-    "asset.rename.dialog.title": 'Rename {type} "{label}"',
+    "asset.rename.dialog.title": 'Modify {type} "{label}"',
```

Comment:
This should apparently be called `asset.modify.dialog.title`, as the translation is not showing properly in the UI when switched to English.

Reply:
See the corresponding constant in `cs.ts`.

Reply:
Oops, missed that one, sorry